test/cases/custom-repos.sh

```diff
@@ -13,6 +13,11 @@ source /etc/os-release
 source /usr/libexec/osbuild-composer-test/set-env-variables.sh
 source /usr/libexec/tests/osbuild-composer/shared_lib.sh
 
+if ! nvrGreaterOrEqual "osbuild-composer" "81"; then
+    echo "INFO: not supported on older versions"
```

Comment:
I think this should be a bit more verbose, Imagine you are me, seeing this in the logs and having no clue what it is referring to :) Maybe something like 'Skipping custom-repos test on osbuild-composer < 81' ?

I also think the commit message should have a short summary. Pointing to the other PR doesn't give me any useful context.